book/plugins/how-to-guides/set-up-development-environment.md

```diff
@@ -145,24 +145,24 @@ conda config --env --set subdir osx-64
 ````
 `````
 
-### Metagenome distribution
+### MOSHPIT (previously known as the *Metagenome distribution*)
 
 `````{tab-set}
 ````{tab-item} macOS
 ```bash
-conda env create -n q2dev-metagenome --file https://raw.githubusercontent.com/qiime2/distributions/dev/latest/passed/qiime2-metagenome-macos-latest-conda.yml
+conda env create -n moshpit-dev --file https://raw.githubusercontent.com/qiime2/distributions/dev/latest/passed/qiime2-moshpit-macos-latest-conda.yml
```

Comment:
```suggestion
conda env create -n q2dev-moshpit --file https://raw.githubusercontent.com/qiime2/distributions/dev/latest/passed/qiime2-moshpit-macos-latest-conda.yml
```
I could go either way on this one, but wondering if it makes sense to keep the same env naming syntax?

Reply:
I thought this way made sense, since the goal has been to put the focus on moshpit, and `dev-moshpit` just felt too weird to me. So I opt to leave this as is if you don't have strong feelings on it.

Reply:
works for me!